hack/terraform/eks/main.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 0.13.1"
+  required_version = ">= 0.13.5"
```

Comment:
Isnt this too old version? can we dump it up

Reply:
Yes, but this means any version greater than 0.13.5 will work. This allows if user want to use an older version of terraform for some reason. We don't require the latest terraform version